elyra/cli/pipeline_app.py

```diff
@@ -162,103 +166,52 @@ def pipeline():
 
 
 @click.command()
-@click.argument('pipeline')
-@click.option('--server-url',
-              required=False,
-              default=None,
-              help='Jupyter Notebook server url including access token')
+@click.argument('pipeline_path')
 @click.option('--runtime',
-              required=False,
-              type=click.Choice(['local', 'kfp', 'airflow'], case_sensitive=False),
-              default='local',
+              required=True,
+              type=click.Choice(['kfp', 'airflow'], case_sensitive=False),
```

Comment:
Should we restrict choices here? I think the processor raises an error if the runtime type isn't supported. If we rely on that instead, we won't have to remember to keep updating this list when we add runtimes. Although, we were talking about detecting the runtime type from the config at some point, so this might not really matter for long